data_and_models/pipelines/sentence_embedding/dvc.yaml

```
@@ -1,4 +1,39 @@
 stages:
+  training_transformers:
+    cmd: >-
+      CUDA_VISIBLE_DEVICES=0,1,2,3
+      python training_transformers/train.py
+      --model_name_or_path clagator/biobert_v1.1_pubmed_nli_sts
+      --train_data_file training_transformers/sentences-filtered_11-527-877.txt
+      --eval_data_file training_transformers/biosses_sentences.txt
+      --line_by_line
+      --mlm
+      --output_dir ../../models/language_modeling/biobert_cord19_v1/
+      --do_train
+      --do_eval
+      --evaluation_strategy steps
+      --per_device_train_batch_size 16
+      --per_device_eval_batch_size 16
+      --logging_dir ../../models/language_modeling__logs
```

```suggestion
      --logging_dir ../../models/language_modeling/biobert_cord19_v1__logs/
```

To follow what is in `build.sh`. Also, to `not` overwrite logs of eventual other language modeling models.